matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/auth/registration/RegistrationWizard.kt

```diff
@@ -73,6 +75,11 @@ interface RegistrationWizard {
      */
     suspend fun dummy(): RegistrationResult
 
+    /**
+     * Perform the other stage.
+     */
```

Comment:
Can you update the comment, to clearly say that it's only for custom registration type. See the doc of [loginCustom](https://github.com/vector-im/element-android/blob/develop/matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/auth/login/LoginWizard.kt#L59) to get inspiration.

Also for parity, maybe rename to `registrationCustom`. Thanks!